modules/porous_flow/test/tests/fluidstate/tests

```diff
@@ -181,6 +181,15 @@
     design = 'PorousFlowWaterVapor.md'
     issues = '#13108'
   []
+  [water_vapor_tabulated]
+    type = 'CSVDiff'
+    input = 'water_vapor_tab.i'
+    csvdiff = 'water_vapor_twophase_tab.csv'
+    threading = '!pthreads'
```

Comment:
Why/when do we skip pthreads? I see a lot of "!pthreads" in the test suite but no reason ever listed. We'd like to note reasons as a comment on the line.

Reply:
it's tradition for porous_flow module
I ll address that in a follow up